### DISCUSSION TITLE
Border that appears on vsplit and split between buffer, which also uses fillchars. How to get rid of this buggy symbols? Screenshot inside

### DISCUSSION THREAD

**Discussion by zayihu** [neovim/neovim#30597] [Q&A]
![image](https://github.com/user-attachments/assets/ceaca699-cbb1-486b-afa8-13009aa626b3)
I want to get rid of symbols on the left. Maybe this is a bug???

**Comment by zeertzjq** [comment]
Set the `vertright` field in 'fillchars'

**Reply by zayihu** [reply]
![image](https://github.com/user-attachments/assets/4a7104ce-8935-4256-bb04-3a5e526c1fcb)
1 is gone with vertright=" ", thank you! Which fill char is for another one?

**Reply by zeertzjq** [reply]
That should be `horizup`

**Reply by zayihu** [reply]
![image](https://github.com/user-attachments/assets/15911d88-8725-4d88-8c90-75b1a284f708)
I tried it, well now is gone, but this brings another problem. How to keep it at the bottom, while hiding in left bottom?

**Reply by zayihu** [reply]
I'm almost there thanks to you!

**Reply by zayihu** [reply]
Maybe if this is possible in current nvim 0.10.1, file this under improvements?

**Reply by zeertzjq** [reply]
You can set the local 'fillchars' value of the window to the left instead of the global value.

**Reply by zayihu** [reply]
Could you please be so kind, and give me example of how to do this. Tried AutoCmds and vim.wo, could not make it work. Could you point me in right direction? 

```
vim.api.nvim_create_autocmd("TermOpen", {
  pattern = "*",
  callback = function()
    vim.wo.fillchars = "vert: ,horiz: "
  end,
})
local function set_fillchars()
  local bufnr = vim.api.nvim_get_current_buf()
  local winid = vim.api.nvim_get_current_win()

  if vim.api.nvim_buf_get_name(bufnr):match("neo-tree") then
    vim.wo[winid].fillchars = "vert: ,horiz: "
  elseif vim.bo[bufnr].buftype == "terminal" then
    vim.wo[winid].fillchars = "vert: ,horiz: "
  end
end

vim.api.nvim_create_autocmd({ "BufEnter", "WinEnter" }, {
  pattern = "*",
  callback = function()
    set_fillchars()
  end,
})
```

How should I do this?

**Reply by zeertzjq** [reply]
Oh wait, there may be something wrong with the current behavior. I'll create an issue: #30633.
